tests/apollo/util/bft.py

```diff
@@ -661,3 +662,15 @@ def force_quorum_including_replica(self, replica_id, primary=0):
                 with trio.move_on_after(interval):
                     if await predicate():
                         return
+
+
+    async def num_of_slow_path(self):
```

Comment:
just for convenience, a method that taking the number of slow path count using the metrics.